pkg/lib/operatorstatus/status.go

```diff
@@ -267,7 +267,7 @@ func findOperatorStatusCondition(conditions []configv1.ClusterOperatorStatusCond
 // RelatedObjects are consumed by https://github.com/openshift/must-gather
 func relatedObjects(name string, opClient operatorclient.ClientInterface, crClient versioned.Interface) ([]configv1.ObjectReference, error) {
 	var objectReferences []configv1.ObjectReference
-	log.Infof("Adding related objects for %v", name)
+	log.Debugf("Adding related objects for %v", name)
```

Comment:
hmm, it looks like everything in here is using the global logger instead of one that's passed in. We should make sure the level is set correctly for this by the init logic [mentioned in an earlier thread](https://github.com/operator-framework/operator-lifecycle-manager/pull/2631/files#r807155143).